_i18n/fr/get-started/contributing.md

```diff
@@ -62,19 +62,19 @@ L'extraction minière garantit que le réseau reste sécurisé et décentralisé
                             <div class="row start-xs">
                                <div class="col-xs-12">
                                 <h3>Faire un don en Monero</h3>
-                                <p>Les donations peuvent être envoyées à donate.getmonero.org ou 44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A</p>
+                                <p>Les contributions peuvent être envoyées à donate.getmonero.org ou 44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A</p>
```

Comment:
"dons" to be consistent with other change above?

Reply:
ok.